moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h

```diff
@@ -576,6 +568,8 @@ class PlanningSceneMonitor : private boost::noncopyable
 
   class DynamicReconfigureImpl;
   DynamicReconfigureImpl* reconfigure_impl_;
+
+  rclcpp::Clock clock_;
```

Comment:
I think this isn't used anywhere

Reply:
It's used multiple times inside `planning_scene_monitor.cpp`